dist/tools/code_in_guides_check/check_for_code.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
Could you add a comment what the overall intention of this file are?

Reply:
sure